src/Rfc6455/Message.php

```diff
@@ -133,4 +135,21 @@ public function isOperation()
     {
         return in_array($this->getFirstFrame()->getOpcode(), [Frame::OP_TEXT, Frame::OP_BINARY]);
     }
+
+    /**
+     * @throws ControlFrameException
+     * @throws TooBigFrameException
+     */
+    public function checkControlFrameSize()
+    {
+        $frame = $this->getFirstFrame();
+
+        if (!$frame->isControlFrame()) {
+            throw new ControlFrameException('It is not a control frame');
+        }
+
+        if ($frame->getPayloadLength() > Frame::MAX_CONTROL_FRAME_SIZE) {
+            throw new TooBigFrameException('A control frame cannot be larger than 125 bytes.');
```

Comment:
This exception is legit but should be throw by the frame itself. (also it should probably be a subclass of `TooBigFrameException` has its more a `TooBigControlFrameException`)